mlflow/gateway/schemas/chat.py

```diff
@@ -22,7 +22,6 @@ class RequestPayload(BaseRequestPayload):
     messages: List[Message] = Field(..., min_items=1)
 
     class Config:
-        extra = Extra.allow
```

Comment:
Does `extra=Extra.ignore` drop extra paremeters dropped in the request payload?

Reply:
Yes, it does

Reply:
Should extra parameters be dropped?

Reply:
How do we set `presence_penalty` for OpenAI chat completions?

Reply:
Nope :). Fixing!

Reply:
Should we be thinking of the RequestModel as abstract and create per-provider instances for the inputs to eliminate trying to coerce all of the optional parameters that each provider has?

Reply:
We should be dropping extra response content and allowing extra request content

Reply:
I don’t think we need to do that right now. I’ll just go ahead and revert these request model changes :). Response models should ignore extras rather than allowing

Reply:
@harupy @BenWilson2 I've updated the PR - can you take another look?